src/lib/Validator/Constraints/EmailInvitation.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\User\Validator\Constraints;
+
+use JMS\TranslationBundle\Model\Message;
+use JMS\TranslationBundle\Translation\TranslationContainerInterface;
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ */
+class EmailInvitation extends Constraint implements TranslationContainerInterface
+{
+    public $message = 'ibexa.user.invitation.user_with_email_exists';
```

Comment:
Something is wrong here. It is more of a translation key. The same string is used in https://github.com/ibexa/user/pull/46/files#diff-7d3421ef4fd83610224c188cb29b40b95eec2cd666cf97eb74b5563dcb394623R25

Reply:
It is translation key. And it is that way as we reuse it in Validator. All of ours Constraints/Validators goes with that pattern.

Reply:
you could ignore this :)